src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -19,22 +21,28 @@
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
-            new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+            new Person(new Name("Alex Yeoh"), new Nric("T0123456A"), new DateOfBirth("2001-01-01"),
+                new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
                 getTagSet("friends")),
-            new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
+            new Person(new Name("Bernice Yu"), new Nric("T0123456B"), new DateOfBirth("2001-01-02"),
+                new Phone("99272758"), new Email("berniceyu@example.com"),
                 new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
                 getTagSet("colleagues", "friends")),
-            new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
+            new Person(new Name("Charlotte Oliveiro"), new Nric("T0123456C"), new DateOfBirth("2001-01-03"),
+                new Phone("93210283"), new Email("charlotte@example.com"),
                 new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
                 getTagSet("neighbours")),
-            new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
+            new Person(new Name("David Li"), new Nric("T0123456D"), new DateOfBirth("2001-01-04"),
+                new Phone("91031282"), new Email("lidavid@example.com"),
                 new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
                 getTagSet("family")),
-            new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
+            new Person(new Name("Irfan Ibrahim"), new Nric("T0123456E"), new DateOfBirth("2001-01-05"),
+                new Phone("92492021"), new Email("irfan@example.com"),
                 new Address("Blk 47 Tampines Street 20, #17-35"),
                 getTagSet("classmates")),
-            new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
+            new Person(new Name("Roy Balakrishnan"), new Nric("T0123456AF"), new DateOfBirth("2001-01-06"),
```

Comment:
Hi would like to mention since I encountered the error just now in case anyone was facing any problems but the input for NRIC here is wrong, there's 2 letters at the end.